components/Author.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+
+import AuthorCollection from '../interfaces/AuthorCollection'
+import { MarkdownFileWithUrl } from '../pages/authors'
+
+import ContentCardList from './ContentCardList'
+import PageLayout from './PageLayout'
+
+export interface Props {
+ validAuthor: AuthorCollection
+ filteredRecordsWithUrl: MarkdownFileWithUrl[]
+}
+
+const Author: React.FunctionComponent<Props> = props => {
+  const authorName = props.validAuthor.name
+  return (
+      <PageLayout documentTitle={` ${authorName}`} appendSiteTitle={true}>
+         <div className="mb-4 w-100 p-2 d-flex flex-column">
+            <div className=" w-25 align-self-center p-3">
+              {props.validAuthor.image ?
+              <img src={props.validAuthor.image} alt="Author Avatar" className="card-img-top rounded-circle" /> :
+              <img src="/authors/unisex-avatar.svg" alt="Author Avatar" className="card-img-top rounded-circle  p-2" />}
```

Comment:
We don't have avatar images yet, so I'm not sure if we should show the default one for every author - or hide the `<img`> entirely for now? We can bring the `<img>` back when we have actual avatars added. This is just a thought, I don't feel strongly about the avatars either way.

Reply:
I see an opportunity to improve this by first determining the author image or default, and then using it in a single instance of an `<img>` tag.
```ts
const authorImage = props.author.image ?? "/authors/unisex-avatar.svg"
```

```suggestion
              <img src={authorImage}{ alt="Author Avatar" className="card-img-top rounded-circle p-2" />}
```

Reply:
We don't necessarily need avatar image functionality at this point, either, though I agree that 

```
const authorImage = props.author.image ?? "/authors/unisex-avatar.svg"
```

is cleaner handling. 

As an aside, would prefer not to have a "unisex" avatar as that usually defaults to male. Maybe we can have an abstract image instead.

Reply:
@ltagliaferri what do you think about removing images completely for now till we have them?

Reply:
Yes, you can remove